administrator/language/en-GB/com_cookiemanager.ini

```diff
@@ -0,0 +1,53 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_COOKIEMANAGER="Cookie Manager"
+COM_COOKIEMANAGER_ACCEPT_BUTTON_TEXT="Accept all cookies"
+COM_COOKIEMANAGER_CATEGORIES="Categories"
+COM_COOKIEMANAGER_CONFIGURATION="Cookie Manager: Options"
+COM_COOKIEMANAGER_CONSENT_BUTTON_TEXT="Cookie consent"
+COM_COOKIEMANAGER_COOKIE_DESCRIPTION="This website uses cookies to ensure that we give you the best experience on our website."
+COM_COOKIEMANAGER_COOKIE_DESC_HINT="It registers a unique ID by the provider that is used to generate statistical data on how the visitor uses the website."
+COM_COOKIEMANAGER_COOKIE_DESC_LABEL="Enter a short description explaining the advantage for the user"
+COM_COOKIEMANAGER_COOKIE_EXP_PERIOD_DAYS="Day(s)"
```

Comment:
```suggestion
COM_COOKIEMANAGER_COOKIE_EXP_PERIOD_DAYS="Days"
```

Reply:
I have written like this because it may be 1 day or 2 days so day(s)
but if it is wrong I will change at all 7 places

Reply:
It looks ugly and might be more ugly in other languages. As you select an engineering unit, so to say, the plural is sufficient. It would be different if you would combine that in language sting, like "Expiration period = 1 second", "Expiration period = 2 seconds", then it would need to use the plural functionality of our language system. But here you have first the dropdown and then the value in separate fields, so it doesn't need to make singular plural things.

Reply:
Ok  understood

Reply:
Joomla's language implementation supports cardinality.

Reply:
@nibra Yes but that makes no sense for the lis field's options discussed here.